src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationRefreshOptions.cs

```diff
@@ -14,7 +14,18 @@ public class AzureAppConfigurationRefreshOptions
     {
         internal TimeSpan CacheExpirationInterval { get; private set; } = RefreshConstants.DefaultCacheExpirationInterval;
         internal ISet<KeyValueWatcher> RefreshRegistrations = new HashSet<KeyValueWatcher>();
-        
+        internal bool RefreshUpdatesOnly { get; private set; } = false;
+
+        /// <summary>
+        /// Register the marked as updated key-values to be refreshed when the configuration provider's <see cref="IConfigurationRefresher"/> triggers a refresh.
+        /// The <see cref="IConfigurationRefresher"/> instance can be obtained by calling <see cref="AzureAppConfigurationOptions.GetRefresher()"/>.
+        /// </summary>
+        public AzureAppConfigurationRefreshOptions RegisterUpdatesOnly()
```

Comment:
Does this actually do something, or could just also just omit the ConfigureRefresh() ?